sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -243,37 +476,79 @@ def add_policy_management_certificate(self, certificate_to_add, signing_key, **k
         for the certificate is the SHA1 hash of the DER encoding of the
         certificate.
 
+        .. admonition:: Example: Generating and adding a new policy management
+            certificates for an isolated attestation instance.
+                    
+            .. literalinclude:: ../samples/sample_get_set_policy.py
+                :start-after: [BEGIN add_policy_management_certificate]
+                :end-before: [END add_policy_management_certificate]
+                :language: python
+                :dedent: 12
+                :caption: Adding a policy management certificate.
+
         """
-        key=JSONWebKey(kty='RSA', x5_c = [ base64.b64encode(certificate_to_add).decode('ascii')])
-        add_body = AttestationCertificateManagementBody(policy_certificate=key)
-        cert_add_token = AttestationToken[AttestationCertificateManagementBody](
+        signing_key = kwargs.pop('signing_key', None)
+        signing_certificate = kwargs.pop('signing_certificate', None)
+        if not signing_key:
+            signing_key = self._signing_key
+        if not signing_certificate:
+            signing_certificate = self._signing_certificate
```

Comment:
Unfortunately ._signing_key is a RSAKey, signing_key is PEM encoded.

Reply:
The re-use of the variable is ... unfortunate, but it's tied to the fact that the signing_key keyword arg to AttestationToken is polymorphic - it can either be a str or bytes or it can be a cryptography object. 

Which means that your suggestion works just fine, but it's confusing so I added a comment explaining what is going on.